api/v1/routes/blog.py

```diff
@@ -15,3 +19,11 @@ def get_all_blogs(db: Session = Depends(get_db)):
         return []
     return blogs
 
+
+@blog.get("/{id}", response_model=SingleResponse)
+def get_blog_by_id(id: str, db: Session = Depends(get_db)):
+    blog_post = blog_service.fetch(db, id)
+    if not blog_post:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND, detail="Blog post not found.")
+    return blog_post
```

Comment:
wrong response format

Reply:
I will work on that now, thank you

Reply:
@zxenonx ... I have implemented the change the correct the response format.
Thank you.